docs/examples/index.md

```diff
@@ -1,8 +1,138 @@
 ---
-title: Basic usage
-sidebar_label: Basic usage
+sidebar_label: Examples
+title: Examples
 ---
 
-This section contains examples of how to use Apify Client for Javascript.
+# Code examples
 
-There is nothing here yet.
+## Different ways how to pass an input
+
+The fastest way to get results from an Actor is to pass input directly to the `call` function.
+
+```javascript
+import { ApifyClient } from 'apify-client';
+
+// Client initialization with the API token
+const client = new ApifyClient({ token: 'MY_APIFY_TOKEN' });
+
+const actorClient = client.actor('apify/instagram-hashtag-scraper');
+
+const input = { hashtags: ['twitter'], resultsLimit: 20 };
+
+// Run the actor and wait for it to finish up to 60 seconds. Input is not persisted for next runs.
+const runData = await actorClient.call(input, { waitSecs: 60 });
+```
+
+To run multiple inputs with the same Actor, most convenient way is to create multiple [tasks](https://docs.apify.com/platform/actors/running/tasks) with different inputs.
+Task input is persisted on Apify platform when task is created.
+
+```javascript
+import { ApifyClient } from 'apify-client';
+
+// Client initialization with the API token
+const client = new ApifyClient({ token: 'MY_APIFY_TOKEN' });
+
+const socialsHashtags = ['facebook', 'twitter', 'instagram'];
+
+// Multiple input schemas for one Actor can be persisted in tasks. Tasks are saved in the Apify platform and can be run multiple times.
+const socialsTasks = socialsHashtags.map((hashtag) => client.tasks().create({
```

Comment:
`socialsTasks` is array of promises, not tasks, it would be better to adjust the name so its clear what it holds